cli/pom.xml

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (c) 2018 Contributors to the Eclipse Foundation
+
+ See the NOTICE file(s) distributed with this work for additional
+ information regarding copyright ownership.
+
+ This program and the accompanying materials are made available under the
+ terms of the Eclipse Public License 1.0 which is available at
+ https://www.eclipse.org/legal/epl-v10.html
+
+ SPDX-License-Identifier: EPL-1.0
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-bom</artifactId>
+        <version>0.7-M2-SNAPSHOT</version>
+        <relativePath>../bom</relativePath>
+    </parent>
+
+    <artifactId>hono-cli</artifactId>
+    <name>Hono command-line interface</name>
+    <url>https://www.eclipse.org/hono</url>
+
+    <description>Hone Command line Interface</description>
+
+    <properties>
+        <docker.skip.build>true</docker.skip.build>
+        <hono.messaging.host>hono-service-messaging.hono</hono.messaging.host>
+        <hono.registration.host>hono-service-device-registry.hono</hono.registration.host>
+        <hono.auth.host>hono-service-auth.hono</hono.auth.host>
+        <!--
+          The shared secret to use for creating/validating registration assertions.
+         -->
+        <hono.regAssertion.sharedSecret>g#aWO!BUm7aj*#%X*VGXKFhxkhNrMNj0</hono.regAssertion.sharedSecret>
```

Comment:
none of these properties except for `docker.skip.build` are relevant to the CLI module. Please remove ...